projects/plugins/jetpack/modules/masterbar/admin-menu/class-jetpack-admin-menu.php

```diff
@@ -47,19 +29,16 @@ public function is_rtl() {
 	public function reregister_menu_items() {
 		global $menu, $submenu;
 
-		// Change the menu only when rendered in Calypso.
-		if ( $this->is_api_request || ( defined( 'REST_REQUEST' ) && REST_REQUEST ) ) {
-			// Reset menus so there are no third-party plugin items.
-			$menu    = array(); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
-			$submenu = array(); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+		// Reset menus so there are no third-party plugin items.
+		$menu    = array(); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+		$submenu = array(); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
 
-			parent::reregister_menu_items();
+		parent::reregister_menu_items();
 
-			$this->add_feedback_menu();
-			$this->add_wp_admin_menu();
+		$this->add_feedback_menu();
+		$this->add_wp_admin_menu();
```

Comment:
Should these lines be indented with one more tab?

Reply:
Why? There is no `if` statement now, so everything can be placed at the "root":

<img width="836" alt="Screen Shot 2021-06-07 at 11 21 37" src="https://user-images.githubusercontent.com/1233880/120992200-a7cf7900-c782-11eb-8a58-09366d0da194.png">